glsl.cc

```diff
@@ -490,42 +528,42 @@ float getSnowMountainHeight(vec2 position)
 float getMountainHillsHeight(vec2 position)
 {
     // defining terrain height parameters
-    const float highMountainsHeight = MAX_TERRAIN_HEIGHT;
-    const float lowMountainsHeight = MAX_TERRAIN_HEIGHT / 2.f;
-    const float smallHillsHeight = MAX_TERRAIN_HEIGHT / 3.f;
+    const float TALL_MOUNTAIN_HEIGHT = MAX_TERRAIN_HEIGHT / 3.0;
+    const float SHORT_MOUNTAIN_HEIGHT = MAX_TERRAIN_HEIGHT / 6.0;
+    const float SHORT_HILLS_HEIGHT = MAX_TERRAIN_HEIGHT / 12.0;
```

Comment:
Are we actually using TALL_MOUNTAIN_HEIGHT or SHORT_HILLS_HEIGHT? Looks like we're maybe only using SHORT_MOUNTAIN_HEIGHT.

Reply:
We're using both of them in the function ! 
it's a long function tho, so now that you mentioned this, I might try to break it into smaller ones